rust/src/edge.rs

```diff
@@ -180,23 +184,28 @@ impl SeleniumManager for EdgeManager {
                 Ok(driver_version)
             }
             _ => {
-                if browser_version.is_empty() {
-                    let latest_stable_url = format!("{}{}", DRIVER_URL, LATEST_STABLE);
-                    self.log.debug(format!(
-                        "Reading {} latest version from {}",
-                        &self.driver_name, latest_stable_url
-                    ));
-                    let latest_driver_version = read_version_from_link(
-                        self.get_http_client(),
-                        latest_stable_url,
-                        self.get_logger(),
-                    )?;
-                    browser_version = self.get_major_version(latest_driver_version.as_str())?;
-                    self.log.debug(format!(
-                        "Latest {} major version is {}",
-                        &self.driver_name, browser_version
-                    ));
-                }
+                let browser_version = match browser_version {
+                    Some(version) => version.to_string(),
+                    _ => {
+                        let latest_stable_url = format!("{}{}", DRIVER_URL, LATEST_STABLE);
```

Comment:
Since have changed this formatting, please do it consistently. For instance, this one should be:

```
format!("{DRIVER_URL}{LATEST_STABLE}");
```

Please review this in the rest of modules.

Reply:
That was done in commit e9c349911ab16c8f444f159650929c62a52ddb98.